nest-city-account/src/utils/subservices/tax.subservice.ts

```diff
@@ -56,15 +59,19 @@ export class TaxSubservice {
     }
   }
 
-  async loadDataFromNoris(data: RequestPostNorisLoadDataDto) {
+  async loadDataFromNoris(
+    data: RequestPostNorisLoadDataDto
+  ): Promise<AxiosResponse<CreateBirthNumbersResponseDto>> {
```

Comment:
Is there any reason to introduce `Promise<AxiosResponse<CreateBirthNumbersResponseDto>>` ? The great thing about TS it will autoderive the type and I find it to be a anti-pattern in this case.

I think this video explains it well: https://www.youtube.com/watch?v=nwSe95uFN8E

> Your default attitude towards return types should be 'don't use them', unless one of three things are true:
> 
> You're writing a function with multiple branches
> You're building a library
> You have a specific TS perf concern and you're banging your head against the wall

Please consider reverting it back.